www/src/components/shell/terminal/actionbar/MoreOptions.tsx

```diff
@@ -0,0 +1,406 @@
+  const [gitUrl, setGitUrl] = useState<{name: string, organization: string}>()
+
+  const close = useCallback(() => {
+    setOpen(false)
+    onClose()
+  }, [onClose])
+
+  useEffect(() => {
+    if (!shellConfiguration) return
+
+    setGitUrl(GitUrlParse(shellConfiguration.git.url))
+  }, [shellConfiguration])
+
+  return (
+    <Modal
+      size="large"
+      open={open}
+      onClose={close}
+      style={{ padding: 0 }}
+    >
+      <Flex
+        direction="column"
+        gap="large"
+      >
+        <Span
+          body2
+          color="text-xlight"
+        >CLOUD SHELL INFO
+        </Span>
+        <Span body1>The Plural-hosted cloud shell allows you to access your cluster’s git repository without syncing locally. </Span>
+        <Flex
+          direction="column"
+          gap="medium"
+        >
+          <FormField
+            label="GitHub organization"
+            hint={(
+              <Span
+                caption
+                color="text-xlight"
+              >
+                <A
+                  inline
+                  href="https://www.plural.sh/contact"
+                  target="_blank"
+                >Contact us
+                </A>
+                &nbsp;for support if you need to change the Git repo owner.
+              </Span>
+            )}
+          >
+            <Input
+              placeholder="organization"
+              value={gitUrl?.organization}
+              disabled
+            />
+          </FormField>
+
+          <FormField label="GitHub repo">
+            <Input
+              placeholder="repository"
+              value={gitUrl?.name}
+              disabled
+            />
+          </FormField>
+        </Flex>
+        <Flex justify="flex-end">
+          <Button onClick={close}>Okay</Button>
+        </Flex>
+      </Flex>
+    </Modal>
+  )
+}
+
+// TODO: Finish once API for shell update is added
+function CloudCredentialsModal({ onClose }) {
+  const [open, setOpen] = useState(true)
+  const close = useCallback(() => {
+    setOpen(false)
+    onClose()
+  }, [onClose])
+
+  return (
+    <Modal
+      size="large"
+      open={open}
+      onClose={close}
+      style={{ padding: 0 }}
+    >
+      <Flex
+        direction="column"
+        gap="large"
+      >
+        <Span
+          body2
+          color="text-xlight"
+        >UPDATE CLOUD CREDENTIALS
+        </Span>
+
+        <Flex direction="column">
+          <Flex direction="column">
+            <FormField label="Cloud provider">
+              <Input placeholder="provider" />
+            </FormField>
+
+            <FormField label="Cloud provider">
+              <Input placeholder="provider" />
+            </FormField>
+
+            <FormField label="Cloud provider">
+              <Input placeholder="provider" />
+            </FormField>
+
+            <FormField label="Cloud provider">
+              <Input placeholder="provider" />
+            </FormField>
```

Comment:
4 times the same? It is not finished?

Reply:
Yes, there is a TODO on the top of this component, and the button is not shown in the UI.

Reply:
Ok, just wanted to check.